src/connectors/ConnectorUpgradeableBase.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.17;
+
+import {OwnableUpgradeable} from "openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+import {Constants} from "../lib/Constants.sol";
+
+abstract contract ConnectorUpgradeableBase is OwnableUpgradeable, UUPSUpgradeable {
+    /// gap for upgrade safety <- can be used to add new storage variables(using up to 49  32 byte slots) in new versions of this contract
+    /// If used, decrease the number of slots in the next contract that inherits this one(ex. uint256[48] __gap;)
+    uint256[49] __gap;
```

Comment:
Since we're extending this base class in all the others and this gap is going to be at the beginning of the storage it doesn't make sense to keep it, right? @kstdl 
There won't really be a scenario where we want to add something in this base contract that adds into all the other contracts and for the other contracts we'll just add it to the end of the state list.